edx_exams/apps/api/v1/views.py

```diff
@@ -123,7 +123,7 @@ def handle_exams(cls, request_exams_list, course_exams_qs, course_id):
                 provider = None
                 # get exam type class, which has specific attributes like is_proctored, is_timed, etc.
                 exam_type_class = get_exam_type(exam['exam_type'])
-                config = CourseExamConfiguration.objects.filter(course_id=course_id).first()
+                config = CourseExamConfiguration.get_configuration_for_course(course_id)
```

Comment:
Question: I see that this approach feels more readable (i.e. using `get_configuration_for_course` method instead of `filter`), but is there any other reason to use this approach rather than filter? Is filter more prone to bugginess here? Just curious. :)

Reply:
mostly just about creating a reusable function for a commonly used database operations. As far as using get() under the hood instead of filter() the query will only ever return zero or one objects since course_id is unique at the database level. IMO it's more appropriate then returning the first element in a list of 1

Reply:
That makes sense to me! thanks for clarifying.